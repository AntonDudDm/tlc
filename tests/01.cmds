sleep 1; echo foo; sleep 2; echo bar; sleep 1
