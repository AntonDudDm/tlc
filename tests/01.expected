0 files...          1 files...          2 files...  
